pkg/utils/erofs/erofs_linux.go

```diff
@@ -15,10 +15,12 @@ import (
 func Mount(bootstrapPath, fsID, mountPoint string) error {
 	mount := unix.Mount
 
-	opts := "fsid=" + fsID
+	opts := fmt.Sprintf("domain_id=%s,fsid=%s", "global-domain", fsID)
```

Comment:
It's better to make "global-domain" as a global constant

Reply:
Thanks, I'll revise it.